snewpdag/data/uniform.csv

```diff
@@ -1,8 +1,8 @@
 "Control","Pass",,"'line’: 100",,
 "Series-new","ops.NewTimeSeries","Control","'out_field':'timeseries','start':(186,321)"
 "Hist-new","ops.NewTimeHist","Control","'out_field':'timehist','start':(184,321),'duration':5,'nbins':100"
-"Uniform-series","gen.Uniform","Series-new","'field':'timeseries','rate':50,'duration':10"
-"Uniform-hist","gen.Uniform","Hist-new","'field':'timehist','rate':100,'duration':5"
+"Uniform-series","gen.Uniform","Series-new","'field':'timeseries','rate':50,'tmin':0,'tmax':10"
+"Uniform-hist","gen.Uniform","Hist-new","'field':'timehist','rate':100,'tmin':0,'tmax':5"
```

Comment:
1. rate here is in Hz or is events between tmin and tmax?
2. for the histogram case, the Nbins between min and max or the bin size should be added in the fields (but maybe is comes before the uniform generator and is not passed again here?)

Reply:
Yes, rate is in seconds.

Nbins is indeed defined for NewTimeHist.  Uniform only generates random values in the interval and adds them to the object.  (That said, there's a bug:  because of the int64 legacy, Uniform generates integers rather than floats.  Will have to fix that.)